MonoGame.Framework/Content/ContentManager.cs

```diff
@@ -70,6 +70,7 @@ public partial class ContentManager : IDisposable
             'l', // Linux
         };
 
+        private static readonly string[] supportedTexture2DExtensions = new string[] { ".bmp", ".jpg", ".jpeg", ".png" };
```

Comment:
I would order this:  PNG, JPG, JPEG, BMP...  just because PNG is overwhelmingly the most used format in game dev... might as well have it try it first.

BMP is almost never used, so it makes more sense for it to be last.

Reply:
Make sense, I was thinking about the same but in absence of spec I used alphabetic order 😅

Reply:
Thanks for changing it.